docs/pages/workflow/debugging.md

```diff
@@ -4,34 +4,64 @@ title: Debugging
+## Production errors
 
-This stacktrace is **extremely valuable** since it gives you the location the error comes from. For example, in the following clip we know that the error came from the file `LinksScreen.js` on line 10 and column (character) 15.
+Errors or bugs in your production app can be much harder to solve, mainly because you have less context around the error (i.e. where, how, and why did the error occur?). **The best first step in addressing a production error is to reproduce it locally.** Once you reproduce an error locally, you can follow the [development debugging process](#development-errors) to isolate and address the root cause.
 
-<Video file="debugging/stacktrace.mp4" />
+> **Hint**: sometimes, running your app in "production mode" locally will show errors that normally wouldn't be thrown. You can run an app locally in production by running `expo start --no-dev --minify`. "--no-dev" tells the server not to be run in development mode, and "--minify" will minify your code the same way it is for production Javascript bundles.
+
+Using an automated error logging system like [Sentry](../../guides/using-sentry/) is a huge help in identifying, tracking, and resolving Javascript errors in your production app. This will give you a good sense of how many people are running into an error, how often, when, **and it even provides sourcemaps so you will have stacktraces of your errors!** Sentry is one of those tools that if you wait until you need it to install it, then you waited too long. Also- Sentry is free up to 5000 events/month.
+
+### My production app is crashing
+
+This can be a really frustrating scenario, since it gives you very little information to go off of on first glance. But, in reality, crashes can be one of the easiest-to-solve errors once you:
+
+- [Access the native device logs](../logging/#optional-manually-access-device-logs)
+- Reproduce the crash (either using your production app, or the Expo client app)
+- Search the logs for a "fatal exception" (there could be a few) to see exactly what is causing your app to crash
+
+With that information, you should be able to identify where the error is coming from, or at least search the internet for possible causes & solutions.
+
+### My app crashes on certain (older) devices
+
+This indicates a performance issue. You likely need to run your app through a profiler to get a better idea of what processes are killing the app, and [React Native provides some great documentation for this](https://reactnative.dev/docs/profiling). We also recommend using [React Devtools](https://www.npmjs.com/package/react-devtools) and the included profiler, which makes it super easy to identify performance sinks in your app.
```

Comment:
rather than "This indicates" maybe "This might indicate that there is a performance issue"